### ISSUE TITLE
Clarify documentation of start and end in TSV spec

### ISSUE THREAD

**Issue by frederik-elwert** [LinkedPasts/linked-places#16]
The documentation of `start` currently reads:

> Earliest relevant date in ISO 8601 form (YYYY-MM-DD); …

For me, “relevant date” seems to be a bit vague. E.g., does this date refer to the use of the *name* given, or to the existence/use of the *place*? Of course the TSV is less expressive than the JSON-LD format, but it would be helpful to understand how a date given here is interpreted during import.

**Comment by kgeographer** [comment]
Good point, thanks. I've [updated the spec](https://github.com/LinkedPasts/linked-places/blob/master/tsv_0.2.md). How does it read now?

**Comment by frederik-elwert** [comment]
Yes, that’s very helpful! (Sorry for the late feedback, somehow the notification got lost.)
